packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -259,6 +259,45 @@ void main() {
     });
   });
 
+  testWidgets('picker honours showDayOfWeek supported only in date mode', (WidgetTester tester) async {
```

Comment:
A little confusing to read. Maybe:

```suggestion
  testWidgets('showDayOfWeek is supported only in date mode', (WidgetTester tester) async {
```